### ISSUE TITLE
compareSpectra function error

### ISSUE THREAD

**Issue by Scottyoungabc** [rformassspectrometry/Spectra#218]
I was trying to using massbank backend in Spectra after data preprocessing by MSnbase packages. After using "chromPeakSpectra" with "return.type=spectra", the error popped up like this. felt like the compare functions in both packages went into a collision? May I know any way to use the specific compareSpectra function in Spectra?
`> res <- compareSpectra(mbank_sub,query_spectra,ppm=20)
Error in (function (classes, fdef, mtable)  : 
  unable to find an inherited method for function ‘compareSpectra’ for signature ‘"Spectra", "Spectra"’`

**Comment by jorainer** [comment]
Yes, `compareSpectra` is defined both in `MSnbase` and `Spectra` - you could explicitely call the one from `Spectra` using `res <- Spectra::compareSpectra(mbank_sub, query_spectra, ppm = 20)`.

I've also just finished a new (short) tutorial of using the `MetaboAnnotatio` package we're currently developing for MS/MS spectra matching including its use in an `xcms`-based analysis workflow. It's included in the [SpectraTutorials](https://jorainer.github.io/SpectraTutorials/) page (the [MS/MS Spectra Matching with MetaboAnnotation](https://jorainer.github.io/SpectraTutorials/articles/Spectra-matching-with-MetaboAnnotation.html) article).

**Comment by YangjjMSresearch** [comment]
btw, I enjoyed your workshop on youtube a lot. Thanks a lot for your help!
